pandas/core/generic.py

```diff
@@ -6362,6 +6362,16 @@ def fillna(
         1   3.0 4.0 NaN 1
         2   NaN 1.0 NaN 5
         3   NaN 3.0 NaN 4
+
+        Replacement happens along same column names and same indices
```

Comment:
Can you write this as "When replacing with a DataFrame, replacement happens along the same column names and same indices:" (may need to split it across two lines). Other than that, looks good, thanks!

Reply:
It has been updated, please take a look